src/app/public/src/modules/analytics/analytics.component.html

```diff
@@ -0,0 +1,4 @@
+<!-- Google Tag Manager -->
+<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-W56QP9"
```

Comment:
This will never run. To test: disable JavaScript in your browser and type `skyux serve`. The `<noscript>` tag won't be added to the markup.

We can probably kill this template altogether and make this into a directive `stacheGoogleAnalytics`.